requirements.txt

```diff
@@ -0,0 +1 @@
+numpy
```

Comment:
I think, we do not need it as it is not present inside setup.pu

Reply:
We do need numpy. Added it to setup.py